src/frontend/apps/desk/src/features/teams/components/PanelTeam.tsx

```diff
@@ -18,15 +18,16 @@ export const PanelTeam = ({ team }: TeamProps) => {
 
   const commonProps = {
     className: 'p-t',
-    width: 36,
+    width: 52,
     style: {
       borderRadius: '10px',
+      flexShrink: 0,
     },
   };
 
   return (
     <Box as="li" $direction="row" $align="center" $gap="0.5rem">
-      {team.accesses.length ? (
+      {team.accesses.length > 1 ? (
```

Comment:
[216f184](https://github.com/numerique-gouv/people/pull/43/commits/216f1842596371fe67ad85a880c55c71ba66ccb9)

Reply:
Let's discuss it orally, I mean we should check if it has one access, and this access is with the role owner

Reply:
can be improved in a next PR btw

Reply:
As from our discussions, a team
- cannot be empty
- has every time at least 1 owner

If the team is empty, it will be impossible to appear in the panel. It shouldn't be necessary to do another check on the front side, or the back is missing something.